DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/sample.json

```
@@ -0,0 +1,71 @@
+{
+  "guid": "CE509751-B84A-44D1-A391-2007937F3A7F",
+  "name": "MMHost",
+  "categories": ["Toolkit/oneAPI Direct Programming/C++SYCL FPGA/Tutorials/Features/experimental"],
+  "description": "An Intel® FPGA tutorial demonstrating the usage of mmhost macros",
+  "toolchain": ["icpx"],
+  "os": ["linux", "windows"],
+  "targetDevice": ["FPGA"],
+  "builder": ["ide", "cmake"],
+  "languages": [{"cpp":{}}],
+  "commonFolder": {
+    "base": "../../../..",
+    "include": [
+      "README.md",
+      "Tutorials/Features/experimental/mmhost",
+      "include"
+    ],
+    "exclude": []
+  },
+  "ciTests": {
+    "linux": [
+      {
+        "id": "fpga_emu",
+        "steps": [
+          "icpx --version",
+          "mkdir build",
+          "cd build",
+          "cmake ..",
+          "make fpga_emu",
+          "./mmhost.fpga_emu"
+        ]
+      },
+      {
```

```suggestion
      {
        "id": "fpga_emu_2",
        "steps": [
          "icpx --version",
          "mkdir build",
          "cd build",
          "cmake ../part2_single_host",
          "make fpga_emu",
          "./mmhost.fpga_emu"
        ]
      },
      {
```
add ciTests for each of the parts as shown in the [fpga_compile code sample](https://github.com/HakimTem/oneAPI-samples/blob/8cbc4fffb2f53cc95e5df743b36fe9e6cffdad79/DirectProgramming/C%2B%2BSYCL_FPGA/Tutorials/GettingStarted/fpga_compile/sample.json)